packages/gestalt/src/Tabs.js

```diff
@@ -153,6 +153,7 @@ function TabV2({
   index,
   isActive,
   onChange,
+  ref,
```

Comment:
same here. I'm  not sure if ref is passed... 
![image](https://user-images.githubusercontent.com/10593890/118894894-2008f400-b8d3-11eb-973e-d7aaa529c432.png)

Reply:
yep, you're right 😬😅